src/lib/fast-get.c

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
not sure src/lib is a suitable location for it. Looking at it, it contains mostly "legacy" files, there's also a comment in zephyr/CmakeLists.txt:
```
	# SOF library - parts to transition to Zephyr over time
```
@kv2019i what would be a good place for it? zephyr/lib?

Reply:
ack, lets put in zephyr/lib